### PR TITLE
fix: windows not working

### PR DESCRIPTION
**Pull request by userquin**

### CODE REVIEW

File: src/node/index.ts [hunk 1]
```diff
@@ -45,16 +49,35 @@ function PluginInspect(options: Options = {}): Plugin {
...
       plugin.transform = async function(this: any, ...args: any[]) {
-        const code = args[0]
-        const id = args[1]
+        const resolvedArgs = resolveArguments(true, args)
+        const code = resolvedArgs[0]
+        const id = resolvedArgs[1]
+
         const start = Date.now()
-        const _result = await _transform.apply(this, args as any)
+        const _result = await _transform.apply(this, resolvedArgs as any)
```
Comment (antfu) [inline src/node/index.ts:80]: I still don't think we should mutate the args passing to plugins.

File: src/node/index.ts [hunk 2]
```diff
@@ -45,16 +49,35 @@ function PluginInspect(options: Options = {}): Plugin {
 
   const transformMap: Record<string, TransformInfo[]> = {}
   const idMap: Record<string, string> = {}
+  const windowsIdMap: Record<string, string> = {}
+
+  function resolveArguments(transform: boolean, args: any[]) {
```
Comment (antfu) [inline src/node/index.ts:54]: In this case, I think we could just have it as `normalizeId(id: string)`

File: package.json [hunk 3]
```diff
@@ -38,46 +38,49 @@
   "dependencies": {
     "debug": "^4.3.2",
     "sirv": "^1.0.17",
-    "ufo": "^0.7.9"
+    "ufo": "^0.7.9",
+    "vue": "^3.2.12"
```
[the remark this answers is not in the document]
Comment (userquin) [reply package.json:42]:
```shell
nr dev

> vite-plugin-inspect@0.3.6 dev F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect
> vite src/client

failed to load config from F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\src\client\vite.config.ts
error when starting dev server:
Error: Cannot find module 'vue'
Require stack:
- F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\node_modules\.pnpm\vue-router@4.0.11\node_modules\vue-router\dist\vue-router.cjs.js
- F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\src\client\vite.config.ts
- F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\node_modules\.pnpm\vite@2.5.9\node_modules\vite\dist\node\chunks\dep-05fcddd5.js
- F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\node_modules\.pnpm\vite@2.5.9\node_modules\vite\dist\node\cli.js
- F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\node_modules\.pnpm\vite@2.5.9\node_modules\vite\bin\vite.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (F:\work\projects\quini\GitHub\antfu\vite-plugin-inspect\node_modules\.pnpm\vue-router@4.0.11\node_modules\vue-router\dist\vue-router.cjs.js:10:11)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19)
```

[the remark this answers is not in the document]
Comment (userquin) [reply package.json:42]: ok, I install `vue` as `dev depenedency` and now works.

**Comment by userquin** [comment]
There is no need to add since we can make it working with 2 refresh on the page.

**Pull request closed without merging**